### ISSUE TITLE
crashes when opened

### ISSUE THREAD

**Issue by drift420** [gjedeer/mylocation#17]
It crashes when opening. Running 4.4.2.

**Comment by gjedeer** [comment]
Fixed in 1.3.1: https://github.com/gjedeer/mylocation/issues/15#issuecomment-392256810

**Comment by drift420** [comment]
The build you posted did the same let me get you more info and i will get back to you.

**Comment by gjedeer** [comment]
An ADB log would be useful

**Comment by paulle** [comment]
Can confirm: crashes when opened.Android 8.0 xperia XZ2

**Comment by paulle** [comment]
Logcat

<< log_count = 27 >>>
[05-28 20:04:01.326 11267:11267 I/LoadedApk]
No resource references to update in package alias:android.product.res.overlay.defaultgmessaging

[05-28 20:04:01.326 11267:11267 I/LoadedApk]
No resource references to update in package alias:android.res.overlay_284

[05-28 20:04:01.326 11267:11267 I/LoadedApk]
No resource references to update in package alias:android.res.overlay_305

[05-28 20:04:01.327 11267:11267 I/LoadedApk]
No resource references to update in package alias:android.res.overlay_310_475

[05-28 20:04:01.450 11267:11287 D/OpenGLRenderer]
HWUI GL Pipeline

[05-28 20:04:01.477 11267:11267 D/LogcatReader]
[Logcat] Available buffers: [crash, events, main, radio, system]

[05-28 20:04:01.478 11267:11267 D/LogcatReader]
[Logcat] Default buffers: [main, system, crash]

[05-28 20:04:01.491 11267:11287 I/Adreno]
QUALCOMM build                   : ef3960d, I4ca2995ce0
Build Date                       : 04/05/18
OpenGL ES Shader Compiler Version: EV031.23.00.02
Local Branch                     : 
Remote Branch                    : refs/tags/AU_LINUX_ANDROID_LA.UM.6.3.R1.08.00.00.301.076
Remote Branch                    : NONE
Reconstruct Branch               : NOTHING

[05-28 20:04:01.496 11267:11287 I/Adreno]
PFP: 0x016ee162, ME: 0x00000000

[05-28 20:04:01.500 11267:11287 I/OpenGLRenderer]
Initialized EGL, version 1.4

[05-28 20:04:01.500 11267:11287 D/OpenGLRenderer]
Swap behavior 2

[05-28 20:04:01.525 11267:11267 D/LogcatReader]
[LogcatLiveFragment] onServiceConnected

[05-28 20:04:01.526 11267:11267 D/LogcatReader]
[LogcatLiveFragment] Added all logs

[05-28 20:04:01.527 11267:11267 D/LogcatReader]
[Logcat] onActivityInForeground

[05-28 20:04:01.527 11267:11267 D/LogcatReader]
[Logcat] Posting pending logs

[05-28 20:04:01.671 11267:11272 I/zygote64]
Do partial code cache collection, code=30KB, data=24KB

[05-28 20:04:01.671 11267:11272 I/zygote64]
After code cache collection, code=30KB, data=24KB

[05-28 20:04:01.671 11267:11272 I/zygote64]
Increasing code cache capacity to 128KB

[05-28 20:04:07.193 11267:11267 D/LogcatReader]
[Logcat] onActivityInBackground

[05-28 20:04:32.125 11267:11267 D/LogcatReader]
[Logcat] onActivityInForeground

[05-28 20:04:32.125 11267:11267 D/LogcatReader]
[Logcat] Posting pending logs

[05-28 20:04:32.147 11267:11267 D/LogcatReader]
[LogcatLiveFragment] onServiceConnected

[05-28 20:04:32.437 11267:11272 I/zygote64]
Do partial code cache collection, code=61KB, data=43KB

[05-28 20:04:32.438 11267:11272 I/zygote64]
After code cache collection, code=61KB, data=43KB

[05-28 20:04:32.438 11267:11272 I/zygote64]
Increasing code cache capacity to 256KB

[05-28 20:04:37.625 11267:11267 I/zygote64]
Deoptimizing void android.support.v7.widget.AppCompatTextHelper.updateTypefaceAndStyle(android.content.Context, android.support.v7.widget.TintTypedArray) due to JIT inline cache

[05-28 20:04:37.626 11267:11267 I/zygote64]
Deoptimizing android.graphics.Typeface android.support.v7.widget.TintTypedArray.getFont(int, int, android.support.v4.content.res.ResourcesCompat$FontCallback) due to JIT inline cache

**Comment by paulle** [comment]
With version 1.31 don't crash anymore, but after starting it's frozen and don't get the position (gps enabled).
![screenshot_20180529-142057](https://user-images.githubusercontent.com/4852079/40658973-2ad1c29c-634d-11e8-8cf5-1c5cabb0c7bc.png)

**Comment by gjedeer** [comment]
Glad that 1.3.1 fixed the problem. Re. no updates, isn't it just waiting for a fix? It looks like you're freshly rebooted (because there's no last fix date set, it's retrieved from the OS) - the first fix after reboot usually takes time, regardless of the app.

**Comment by paulle** [comment]
No, the device isn't freshly rebooted (and the gps is working without problems gpstest). Don't know what's going wrong. Device is xperia xz2 with android 8.0.

**Comment by tanukee** [comment]
Can confirm 1.3.1 starts and won't upgrade information except for "last fix: January 1, 1970"
1.3.0 still ok (never had crashes)
Can rule out location in general which is working fine with other apps.
Android 7.1.1 patch level April 1 2018

**Comment by gjedeer** [comment]
Let's see how 1.3.2 works for you guys. @paulle @tanukee

**Comment by tanukee** [comment]
Hi gjedeer, 
phone is running 8.1.0 now but v1.3.1 did not work too.
v1.3.2 is working fine.
Thanks for the fix.

**Comment by paulle** [comment]
v1.3.2 till now isn't on f-droid.
@tanukee from there did you get v1.3.2 ?

**Comment by paulle** [comment]
Meanwhile I got v1.3.2 from f-droid. Working fine, get position. Thank you @gjedeer
